tests/lib/rules/no-case-declarations.js

```diff
@@ -35,9 +35,49 @@ ruleTester.run("no-case-declarations", rule, {
         {
             code: "switch (a) { case 1: { class C {} break; } default: { class C {} break; } }",
             parserOptions: { ecmaVersion: 6 }
+        },
+        {
+            code: `
+                switch (a) { 
+                    case 1: 
+                    case 2: {} 
+                }
+            `,
+            parserOptions: { ecmaVersion: 6 }
```

Comment:
Just noting, in case it wasn't clear, that `parserOptions` is not required here, since no ES6 features are being used.